src/full/Agda/Interaction/Highlighting/Common.hs

```diff
@@ -20,8 +20,7 @@ highlightOnlyCode HighlightAll  _ = False
 highlightOnlyCode HighlightCode _ = True
 highlightOnlyCode HighlightAuto AgdaFileType = False
 highlightOnlyCode HighlightAuto MdFileType   = True
--- TODO: see #3373
-highlightOnlyCode HighlightAuto RstFileType  = False
+highlightOnlyCode HighlightAuto RstFileType  = True
```

Comment:
The functions `highlightOnlyCode` and `highlightedFileExt` are specific to the HTML backend. They should go in `Agda.Interaction.Highlighting.HTML`.

Reply:
Sure! I thought it'll be used somewhere else when I first impl these but seems not.